examples/nips17_adversarial_competition/validation_tool/README.md

```diff
@@ -0,0 +1,22 @@
+
+# Submission validation tool
+
+This tool could be used to verify whether submission file is valid or not.
+It extracts submission, verifies presense and vailidty of metadata and runs
```

Comment:
"submission" -> "the submission"
"presense" -> "presence"
"vailidty" -> "validity"